### ISSUE TITLE
Dimension problem in pytorch_lr_scheduler

### ISSUE THREAD

**Issue by Vikas12911** [aladdinpersson/Machine-Learning-Collection#40]
while performing the model in check accuracy function the input x of size [128,1,28,28].
it should be [128,784].
